__tests__/app.test.js

```diff
@@ -0,0 +1,43 @@
+const app = require("../db/app.js")
+const request = require("supertest")
+const db = require("../db/connection.js")
+const seed = require("../db/seeds/seed.js")
+const data = require("../db/data/test-data")
+
+beforeEach(() => {
+    return seed(data);
+})
+
+afterAll(() => {
+    return db.end();
+})
+
+describe('GET /api/topics', () => {
+    test('1. Returns 200 status code', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+    })
```

Comment:
this test could be included in your test below by expecting the 200 instead of a seperate test.